packages/components/src/border-control/border-control/component.tsx

```diff
@@ -64,10 +65,15 @@ const UnconnectedBorderControl = (
 	} = useBorderControl( props );
 
 	return (
-		<View as="fieldset" { ...otherProps } ref={ forwardedRef }>
+		<View
+			as={ renderAsFieldset ? 'fieldset' : undefined }
+			{ ...otherProps }
+			ref={ forwardedRef }
+		>
 			<BorderLabel
 				label={ label }
 				hideLabelFromVision={ hideLabelFromVision }
+				as={ renderAsFieldset ? 'legend' : undefined }
```

Comment:
I'm thinking that, if this element isn't a `legend`, _and_ if it's hidden from vision, it's essentially useless, because screen readers won't announce it. On the other hand, if this is part of a split control, we _do_ need some form of accessible labelling, otherwise there's no way of telling which side is which.

I think ideally we wouldn't render `BorderLabel` at all unless it's a `legend`, and would instead pass the `label` value directly to the controls below, where it could be prepended to their existing labels. Would this be doable?

Reply:
I'd tend to agree this should either render a fieldset + legend or nothing. Also, if there's a fieldset rendered, the legend (the label prop) should be required, as rendering an unnamed group is a bit pointless. 

> and would instead pass the label value directly to the controls below, where it could be prepended to their existing labels

Not sure that would be ideal, as concatenation in translatable strings is problematic for some languages and should be avoided.